actors/account/tests/account_actor_test.rs

```diff
@@ -27,7 +29,7 @@ fn construction() {
         if exit_code.is_success() {
             rt.call::<AccountActor>(
                 Method::Constructor as MethodNum,
-                &RawBytes::serialize(addr).unwrap(),
+                Some(IpldBlock::serialize_cbor(&addr).unwrap()),
```

Comment:
In `IpldBlock`, I suggest adding a function that returns serialized block already wrapped in `Some`. Or if not, a helper pure function in built-in actors somewhere.

Reply:
Done, thanks for the suggestion.